mesonbuild/backend/ninjabackend.py

```diff
@@ -3130,6 +3125,17 @@ def add_dependency_scanner_entries_to_element(self, target: build.BuildTarget, c
     def get_dep_scan_file_for(self, target: build.BuildTarget) -> str:
         return os.path.join(self.get_target_private_dir(target), 'depscan.dd')
 
+    def _add_deps(self, target: build.BuildTarget, deps: T.Sequence[mesonlib.FileOrString],
+                  to: T.Callable[[str], None]) -> None:
```

Comment:
`to` seems an odd name for a callable... What about something like `add_dep_to` ?